lang/cs/Org.Apache.REEF.Client/Common/JobSubmissionResultImpl.cs

```diff
@@ -120,29 +150,28 @@ internal static string GetTrackingUrl(string filePath)
             return result.Item2;
         }
 
-        internal string GetDriverUrlForYarn(String filePath)
+        internal string GetDriverUrlForYarn(string filePath)
         {
-            _driverUrl = GetTrackingUrl(filePath);
-            return _driverUrl;
+            return GetTrackingUrlAppId(filePath);
```

Comment:
This opens a file written by the Java side of the client, right? How do we make sure that the file exists at this time? What if it wasn't written yet?